exporter/signalfxexporter/internal/translation/translator.go

```diff
@@ -622,7 +622,10 @@ func calculateNewMetric(
 	}
 
 	if tr.Operator == MetricOperatorDivision && *v2 == 0 {
-		logger.Warn(
+		// We can get here if, for example, in the denominator we get multiple
+		// datapoints that have the same counter value, which will yield a delta of
+		// zero.
+		logger.Info(
```

Comment:
Did you mean `Debug` here?
I also not sure if we want to simply mask the message. 
As it was mentioned @jvoravong there could be a flot/int type issue that is causing the denominator to end up as 0

Reply:
Good call out. Did we check that this was not a casting issue?

Reply:
yes, i was just about to push that change 😄

Reply:
If debug levels are enabled then the logger won't be sampled and nothing will functionally change.

Reply:
Regarding this possibly being caused by a cast to int, I don't think it is: the value being tested is a float64

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6e31c3ce927704609fe530fb9ebbcb13b659471f/exporter/signalfxexporter/internal/translation/translator.go#L624-L623

but I tested near-zero denominators just in case I was missing something and the warning didn't come up.